packages/components/src/dropdown/style.scss

```diff
@@ -0,0 +1,3 @@
+.components-dropdown {
+	display: inline-block;
```

Comment:
Thanks for fixing this! Changing the display of the `div` makes sense. Can confirm it's working in Storybook 👍